### ISSUE TITLE
wvhba6470p.com warnings

### ISSUE THREAD

**Issue by StrangeDOS** [magnuschase293/comic_getter#18]
I've been sitting on this issue since I started testing your app.  So this isn't something that just suddenly started happened with the latest changes.

While the app runs and the output shows 'loading', I get warnings from MalwareBytes about malvertising.  Its always the same domain too (wvhba6470p.com).  I don't know if anything can be done about it.  I assume this happens when you are parsing the links or something like that.
![2020-09-18_9-39-44](https://user-images.githubusercontent.com/20542548/93604592-9267b400-f993-11ea-86f6-25f70faa1610.jpg)

**Comment by magnuschase293** [comment]
I don't think it has anything to do with the script. It is related to your security settings on windows. There is a possibility adding support for extensions particularly an adblocker may solve it. I will think about it.

**Comment by StrangeDOS** [comment]
Okay.  In the meantime, I put in an entry in my HOSTS file to redirect any attempt at this domain back to my localhost (so it fails).  No more popups from Malwarebytes :)
